### PR TITLE
[Makefile] require 'install-tools' in default make target

### PR DESCRIPTION
**Pull request by djaglowski**
Potential fix for #12607. See [comment](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/12607#issuecomment-1202685976).

### CODE REVIEW

**Comment by djaglowski** [comment]
@open-telemetry/collector-contrib-approvers, please take a look

**Review by TylerHelmuth — APPROVED** [review]
Approving with the assumption that installing tools should always be a safe operation to perform.

**Pull request merged**
